core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -1125,13 +1125,19 @@ public enum Property {
       "The number of replicas for a table's RFiles in HDFS. When set to 0, HDFS"
           + " defaults are used.",
       "1.3.5"),
+  @Deprecated(since = "2.1.3")
   TABLE_FILE_MAX("table.file.max", "15", PropertyType.COUNT,
-      "The maximum number of RFiles each tablet in a table can have. When"
-          + " adjusting this property you may want to consider adjusting"
-          + " table.compaction.major.ratio also. Setting this property to 0 will make"
-          + " it default to tserver.scan.files.open.max-1, this will prevent a tablet"
-          + " from having more RFiles than can be opened. Setting this property low may"
-          + " throttle ingest and increase query performance.",
+      "This property is currently only used by the deprecated DefaultCompactionStrategy.  This "
+          + "property was used by merging minor compaction in previous versions of Accumulo, "
+          + "but that feature was remove prior to the 2.1.0 release.  The way "
+          + "DefaultCompactionStrategy uses this property could result in a quadratic "
+          + "amount of compaction work happening over time.  For example if a tablets "
+          + "smallest files is 1G and a new 1K files is added, if that new files exceeds "
+          + "this max it would compact the 1G and 1K file which is not ideal.  The ideal "
+          + "way to control the number of tablets per file is by adjusting the "
+          + "table.compaction.major.ratio property because it will perform logarithmic "
+          + "amounts of compaction work over time versus the quadratic amount of work this "
+          + "property can cause. The new DefaultCompactionPlanner does not use this property.",
```

Comment:
```suggestion
      "This property is currently only used by the deprecated DefaultCompactionStrategy. This "
          + "property was used by merging minor compaction in previous versions of Accumulo, "
          + "but that feature was remove prior to the 2.1.0 release. The way "
          + "DefaultCompactionStrategy uses this property could result in a quadratic "
          + "amount of compaction work happening over time. For example if a tablets "
          + "smallest files is 1G and a new 1K files is added, if that new files exceeds "
          + "this max it would compact the 1G and 1K file which is not ideal. The ideal "
          + "way to control the number of tablets per file is by adjusting the "
          + "table.compaction.major.ratio property because it will perform logarithmic "
          + "amounts of compaction work over time versus the quadratic amount of work this "
          + "property can cause. The new DefaultCompactionPlanner does not use this property.",
```

Reply:
I thought I had added a checkstyle rule awhile back to disallow double space after period, to force our string literals to be consistent with only a single space. They often are rendered in HTML, such as in our generated property docs, and since HTML collapses spaces, it can be a real pain to copy and paste something from the HTML property docs and try to grep for it on the command line when it fails to match because of mismatched space counts.

So, I thought I could quickly fix this for you, so we could merge this in, because I thought that was the problem with the build checks. However, I never actually committed that checkstyle rule, and it turns out it wasn't the problem with the build.